jarbas/core/management/commands/companies.py

```diff
@@ -63,17 +72,15 @@ def save_activities(self, row):
 
         return [main], secondaries
 
+
     def serialize(self, row):
```

Comment:
Basically the great point of using `rows` is to get rid of the `serialize` method. The idea is to use the `force_types` keyword argument when calling `rows.import_from_csv` and let `rows` serialize stuff as we need it ([for example](https://github.com/datasciencebr/jarbas/blob/8dfb315ed6affe0026507c8a543c6c3752806cef/jarbas/core/management/commands/reimbursements.py#L37-L49), #211).